packages/analytics/amplify_analytics_pinpoint_dart/lib/src/impl/analytics_client/event_client/queued_item_store/index_db/in_memory_queued_item_store.dart

```diff
@@ -45,4 +46,11 @@ class InMemoryQueuedItemStore implements QueuedItemStore {
 
   @override
   void close() {}
+
+  /// Reset all internal values for next unit test
+  @visibleForTesting
+  void reset() {
```

Comment:
nit: possible to create new instance in the tests instead of adding a public method reseting properties for unit testing?